scaleway/docs-content
# feat: add vale linter

##  The what
The goal is to add [Vale](https://docs.errata.ai/vale/about) prose-linter.

We need to:

- [x] Make sure the tool is what MTA team wants
- [ ] Add more vocabulary into the tool (scaleway-specific words)
- [ ] Add a [GitHub action](https://github.com/errata-ai/vale-action) ?

EDIT: please see https://github.com/scaleway/docs-content/pull/66#issuecomment-912491372 for the last two tasks

## To test
- `brew install vale`
- Make sure the command is well installed -> `vale -h`
- Try the command on a file or folders
`vale ai/index.mdx` -> no errors
`vale ai/` -> errors in `ai/concepts.mdx`
- You can also try [VSCode extension](https://github.com/errata-ai/vale-vscode)
